### ISSUE TITLE
on submit change event is triggered

### ISSUE THREAD

**Issue by Skulli** [RobinHerbots/Inputmask#1392]
Since 3.3.2 an change event is triggered when a form with a masked input is submitted.

```
$(".datepicker-input").inputmask('dd.mm.yyyy');
```

As an example for a used mask and following code to test it.

```
$(":input").bind "change", () ->      
      console.trace()    
```

Its a bit annoying since i am using the change event to show an unsaved warning but that event is making it difficult

**Comment by RobinHerbots** [comment]
@Skulli , 

This behavior was also present in previous versions.  This is expected behavior.

**Comment by Skulli** [comment]
Hmm odd when i downgrade to 3.3.1 i dont get those unsaved form warnings anymore. I gonna test it again.

**Comment by kiesslir** [comment]
Hi,

i upgrade from 3.1.63 to current version (3.3.5-33).

I also use the change event to show an unsaved warning and getting trouble because jquery.inputmask trigger the change event on submit.

Is there a way to omit these?

***update***
Yes, these behavior was also present in 3.1.63, but there the following condition returns FALSE and so the event wasn't triggered:

`undoValue != getBuffer().join("") && $el.change()`

In 3.1.63 both ("undoValue" and "getBuffer().join('')") contained the placeholder (they are equal and therefore the comparison result is FALSE), but in 3.3.5-63 variable "undoValue" is `undefined`

**Comment by RobinHerbots** [comment]
@kiesslir , @Skulli , 

Fixed with #1583
